decorators/headerFooter.js

```diff
@@ -14,15 +14,15 @@ function headerFooter(ComposedComponent) {
       if (!templates || !templates[type]) {
         return null;
       }
-      let className = cx(this.props.cssClasses[type], `ais-${type}`);
+      let cssClass = cx(this.props.cssClasses[type], `ais-${type}`);
```

Comment:
Why renaming to `cssClass`? I try to keep the name of the variables equals to the name of the props they will be bound to, to make it easier to understand